bindings/generator/api/pki.py

```diff
@@ -84,11 +84,21 @@ async def show_certificate_selection_dialog_windows_only() -> Result[
     raise NotImplementedError
 
 
+CUSTOM_TO_RS_STRING_DISPLAY = (
+    "|v| -> Result<_, std::convert::Infallible> { Ok(std::string::ToString::to_string(&v)) }"
+)
```

Comment:
If it's here and not inside the PkiSignatureAlgorithm class because it's generic, it could be moved to common.py